src/librustc/dep_graph/graph.rs

```diff
@@ -1021,25 +1021,17 @@ impl CurrentDepGraph {
     fn complete_anon_task(&mut self, kind: DepKind, task_deps: TaskDeps) -> DepNodeIndex {
         debug_assert!(!kind.is_eval_always());
 
-        let mut fingerprint = self.anon_id_seed;
```

Comment:
Can you add a comment explaining why this works? Especially why it is OK to stable-hash unstable data like `DepNodeIndex` values?

Reply:
Done.